vaadin-grid-theme.html

```diff
@@ -0,0 +1,76 @@
+<link rel="import" href="vaadin-grid-active-item-behavior.html">
+<link rel="import" href="vaadin-grid-selection-behavior.html">
+<link rel="import" href="vaadin-grid-data-source-behavior.html">
+<link rel="import" href="vaadin-grid-row-details-behavior.html">
+
+<dom-module id="vaadin-grid-theme-styles">
+  <template>
+    <style>
+
+    /* Default borders */
+
+    thead tr:last-child th ::content > .cell-content {
+      border-bottom: 1px solid var(--divider-color, rgba(0, 0, 0, 0.08));
```

Comment:
Actually, now that I updated to the latest version, I don’t see the blurry lines anymore. But now there are some gaps between the edges of the grid and the row dividers 😄